README.md

```diff
@@ -0,0 +1,15 @@
+## Welcome to Bucket List ## 
```

Comment:
remove the trailing ##. in markdown syntax, `##` is shorthand for h2, so it should be bold anyway. do this for each of your headers